pages/doc/integrations_tas_custom_app_metrics_howto.md

```diff
@@ -0,0 +1,75 @@
+---
+title: How to Monitor Custom App Metrics on TAS with Tanzu Observability
+keywords:
+tags: [integrations]
+sidebar: doc_sidebar
+permalink: integrations_tas_custom_app_metrics_howto.html
+summary: How to get your custom application metrics to appear in Tanzu Observability by Wavefront.
+---
+
+Out of the box, the Wavefront Nozzle for TAS provides metrics for monitoring the TAS platform. For monitoring applications running **on** the platform, the Nozzle supports two approaches:
```

Comment:
By default, the Tanzu Observability by Wavefront Nozzle provides metrics for monitoring the TAS platform. For monitoring applications that run **on** the platform, the nozzle supports two models: